dev/com.ibm.ws.security.saml.sso_fat.3/publish/servers/com.ibm.ws.security.saml.sso-2.0_fat.3/bootstrap.properties

```diff
@@ -3,7 +3,7 @@ bootstrap.include=../testports.properties
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
 org.apache.xml.security.*=all=enabled:\
 com.ibm.ws.security.saml*=all=enabled:\
-com.ibm.ws.webcontainer.security*=all=enabled
-
+com.ibm.ws.webcontainer.security*=all=enabled:\
```

Comment:
Any reason to leave this on.  That seems like a lot of trace.

Reply:
removed the webcontanier security but leaving the other packages here.